src/shared/locales/en/translation.json

```diff
@@ -63,6 +63,8 @@
         "password": "Password",
         "unrecognisedPassword": "Unrecognised password",
         "unrecognisedPasswordExplanation": "The password was not recognised. Please try again.",
+        "unrecognisedAccount": "Unrecognised account",
+        "unrecognisedAccountExplanation": "Account named {{accountName}} was not recognised. Please go to Advanced Settings reset the wallet.",
```

Comment:
I just realized Advanced Settings isn't accessible if you can't log in. Could you change the last part to `Please reset the wallet.`?

Reply:
Sure, I'll do it